lib/acme/client/resources/account.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+class Acme::Client::Resources::Account
+  attr_reader :url, :status, :contact, :term_of_service, :orders_url
+
+  def self.arguments_from_response(response)
+    attributes = response.body
+    status = attributes.fetch('status')
+    term_of_service = attributes['termsOfServiceAgreed']
+    contact = attributes.fetch('contact', [])
+
+    { term_of_service: term_of_service, status: status, contact: contact }
```

Comment:
Any reason this doesn't return an instance of `Acme::Client::Resources::Account`?

Seems odd to me to have callers get arguments from a class method to then initialize it separately.

Reply:
I want each object to be easy serialize/deserialize.

i.e.; I always want to be able to do `ResourceObject.new(client, ResourceObject.to_h)`

Now that you bring it up, I feel like extracting the argument from the response should live in the client instead of the resource objects.